AutoGraphTests/WebSocketClientTests.swift

```diff
@@ -58,6 +61,83 @@ class WebSocketClientTests: XCTestCase {
         XCTAssertEqual(film?.remoteId, "ZmlsbXM6MQ==")
     }
     
+    func testUnsubscribeRemovesSubscriptions() {
+        let request = try! SubscriptionRequest(request: FilmSubscriptionRequest(), operationName: "film")
+        let subscriber = self.subject.subscribe(request: request, responseHandler: SubscriptionResponseHandler(completion: { _ in }))
+        
+        XCTAssertTrue(self.subject.subscriptions["film}"]!.contains(where: {$0.key == subscriber }))
+        
+        try! self.subject.unsubscribe(subscriber: subscriber)
+        
+        XCTAssertEqual(self.subject.subscriptions.count, 0)
+    }
+    
+    
+    func testUnsubscribeAllRemovesSubscriptions() {
+        let request = try! SubscriptionRequest(request: FilmSubscriptionRequest(), operationName: "film")
+        let subscriber = self.subject.subscribe(request: request, responseHandler: SubscriptionResponseHandler(completion: { _ in }))
+        self.subject.requeueAllSubscribers()
+
+        XCTAssertTrue(self.subject.queuedSubscriptions.contains(where: { $0.key == subscriber }))
+
+        try! self.subject.unsubscribeAll(request: request)
+
+        XCTAssertEqual(self.subject.queuedSubscriptions.count, 0)
```

Comment:
we should make sure the request to close the subscriptions that are active sent out too and they are removed from `subscriptions`